src/content/energy-consumption/index.md

```diff
@@ -7,61 +7,42 @@ sidebar: true
+The ability for any miner to add new blocks only works if there is a cost associated with mining and unpredictability about which specific node submits the next block. These conditions are met by imposing PoW. To be eligible to submit a block of transactions, a miner must be the first to submit the solution to a computationally expensive puzzle. To successfully take control of the blockchain, a dishonest miner would have to consistently win the PoW race by investing in sufficient hardware and energy to outperform the majority of other miners.
 
-![image](energy_use_per_transaction.png)
+This mechanism of securing the network is problematic for several reasons. Miners can increase their odds of success by investing in more and more powerful hardware, creatign conditions for an arms race with miners acquiring increasingly power-hungry mining equipment. This increases the network's energy consumption and generates hardware waste. Ethereum's PoW protocol currently has a total annualized power consumption approximately equal to that of Finland <sup>[^1]</sup> and carbon footprint similar to Switzerland<sup>[^1]</sup>.
 
-<p style="text-align: center;"><small><i>Estimate of PoW energy consumption per tx used in figure based on <a href="https://blog.ethereum.org/2021/05/18/country-power-no-more/" target="_blank" rel="noopener noreferrer">May 2021 data</a>, at time of writing the same source suggested up to <a href="https://digiconomist.net/ethereum-energy-consumption" target="_blank" rel="noopener noreferrer">175.56 Kwh</a></i></small></p>
+PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether cna be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in mroe powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).
```

Comment:
```suggestion
PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether can be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in mroe powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).
```